example/src/example_eco_handler.erl

```diff
@@ -0,0 +1,38 @@
+-module(example_eco_handler).
```

Comment:
The idea for this handler was **not** to always return the same that it gets.
The idea was to store what is PUT in it somewhere… say… in an ets or in application:set_env…
And retrieve that thing on GET